easybuild/easyconfigs/n/nifti2dicom/nifti2dicom-0.4.11-foss-2018b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'nifti2dicom'
+version = '0.4.11'
+
+homepage = 'https://github.com/biolab-unige/nifti2dicom'
+description = """Nifti2Dicom is a conversion tool that converts 3D NIfTI files (and other formats supported by ITK,
+including Analyze, MetaImage Nrrd and VTK) to DICOM. Unlike other conversion tools, it can import a DICOM file that
+is used to import the patient and study DICOM tags, and allows you to edit the accession number and other DICOM
+tags, in order to create a valid DICOM that can be imported in a PACS."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/biolab-unige/nifti2dicom/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['nifti2dicom-0.4.11-no_GUI.patch']
```

Comment:
@crubb Please use the `%(version)s` template here rather than hardcoding `0.4.11`, which serves as a reminder to re-check the patch when bumping the software version.